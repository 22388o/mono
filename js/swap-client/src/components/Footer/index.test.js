import React from 'react';
import { fireEvent, render, screen, waitFor } from '@testing-library/react';
import '@testing-library/jest-dom'; // For the "toBeInTheDocument" matcher
import { Footer } from './index';

describe('Footer component test', () => {
  it('renders component', async () => {

    const {container} = render(
      <Footer />
    );

    expect(container).toMatchSnapshot();
  });
});
