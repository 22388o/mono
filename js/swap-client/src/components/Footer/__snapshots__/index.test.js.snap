// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test renders component 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
  >
    <div
      class="MuiStack-root flex-middle css-m69qwo-MuiStack-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-1eqsa9w-MuiTypography-root"
      >
        v
        0.0.2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1 css-vhekrc-MuiTypography-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="RectangleRoundedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M2 6v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2z"
          />
        </svg>
         All systems ok!
      </p>
    </div>
    <div
      class="MuiStack-root flex-vh-center css-nen11g-MuiStack-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-qq1hiy-MuiTypography-root"
      >
        This is alpha software. Use it under your own risk.
      </p>
    </div>
    <div
      class="MuiStack-root css-y9pb4f-MuiStack-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 flex-middle css-1h9by9k-MuiTypography-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-bq05nv-MuiSvgIcon-root"
          data-testid="HelpOutlineOutlinedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z"
          />
        </svg>
        Help & Support
      </p>
    </div>
  </div>
</div>
`;
